.github/workflows/test-package.yml

```diff
@@ -3,9 +3,9 @@ name: Dart CI
 on:
   # Run on PRs and pushes to the default branch.
   push:
-    branches: [ master ]
+    branches: [ dart-master ]
```

Comment:
Do you want a dart-dev branch instead?

Reply:
It looks like we can't test directly on master on Travis (since we need to build it first). So maybe dart-dev branch seems like a better name. 

I can either create a new PR on dart-dev or you can rename the branch later.

Reply:
https://github.com/dart-lang/ffigen/tree/dart-dev